README.md

```diff
@@ -438,6 +438,16 @@ Extract the two sequences that we want to align. In the commands below, replace
 samtools faidx chr3.fa chr3:186,900,000-187,800,000 >chr3q27.3.fa
 samtools faidx k48/HS0674-contigs.fa 102 | seqtk seq -r >inversion.fa
 ```
+If you do not have "Seqtk" package installed on your system, Install it using:
+
+```sh
+git clone https://github.com/lh3/seqtk.git;
+cd seqtk; make
+```
+and either add add the path, or use it like "seqtk/seqtk". E.g  for the previous exercise:
+```sh
+samtools faidx k48/HS0674-contigs.fa 102 | seqtk/seqtk seq -r >inversion.fa
+```
```

Comment:
Please remove this section and add `seqtk` to the `brew install` list on line 82. (alphabetical order please)